kubernetes/overlays/authorization-updater/dev-refactor/deployment.yaml

```
@@ -32,7 +27,7 @@ spec:
             - name: AUTH_MNGT_SERVICE_URL
```

this has been renamed to (merged right now)
```suggestion
            - name: AUTHORIZATION_MANAGEMENT_URL
```